client/app/email-domain/email-domain.service.js

```diff
@@ -0,0 +1,14 @@
+angular.module('services').service(
+  'EmailDomain',
+  class EmailDomain {
+    constructor(
+      OvhApiEmailDomain,
```

Comment:
According to the staging environment, you will have this issue:

```txt
Error: [$injector:unpr] Unknown provider: OvhApiEmailDomainProvider <- OvhApiEmailDomain <- EmailDomain
```

It means that you have to update [ovh-api-services](https://github.com/ovh-ux/ovh-api-services) I guess.

Reply:
Yes... But I have created a build out of a new branch, that is a replica of this one. In addition, it has the updated dependency to the ovh-api-service changes.

Reply:
Thanks for the update @ganeshkumar1989